wpimath/src/main/java/edu/wpi/first/wpilibj/spline/QuinticHermiteSpline.java

```diff
@@ -80,13 +80,40 @@ protected SimpleMatrix getCoefficients() {
    */
   private SimpleMatrix makeHermiteBasis() {
     if (hermiteBasis == null) {
+      // Given P(i), P'(i), P''(i), P(i+1), P'(i+1), P''(i+1), the control
+      // vectors, we want to find the coefficients of the spline
+      // P(t) = a5 * t^5 + a4 * t^4 + a3 * t^3 + a2 * t^2 + a1 * t + a0.
+      //
+      // P(i)     = P(0)   = a0
+      // P'(i)    = P'(0)  = a1
+      // P''(i)   = P''(0) = 2 * a2
+      // P(i+1)   = P(1)   = a5 + a4 + a3 + a2 + a1 + a0
+      // P'(i+1)  = P'(1)  = 5 * a5 + 4 * a4 + 3 * a3 + 2 * a2 + a1
+      // P''(i+1) = P''(1) = 20 * a5 + 12 * a4 + 6 * a3 + 2 * a2
+      //
+      // [ P(i)     ] = [ 00 00 00 00 00 01 ][ a5 ]
+      // [ P'(i)    ] = [ 00 00 00 00 01 00 ][ a4 ]
+      // [ P''(i)   ] = [ 00 00 00 02 00 00 ][ a3 ]
+      // [ P(i+1)   ] = [ 01 01 01 01 01 01 ][ a2 ]
+      // [ P'(i+1)  ] = [ 05 04 03 02 01 00 ][ a1 ]
+      // [ P''(i+1) ] = [ 20 12 06 02 00 00 ][ a1 ]
+      //
+      // To solve for the coefficients, we can invert the 6x6 matrix and move it
+      // to the other side of the equation.
+      //
+      // [ a5 ] = [ -06.0 -03.0 -00.5  06.0 -03.0  00.5 ][ P(i)     ]
+      // [ a4 ] = [  15.0  08.0  01.5 -15.0  07.0 -01.0 ][ P'(i)    ]
+      // [ a3 ] = [ -10.0 -06.0 -01.5  10.0 -04.0  00.5 ][ P''(i)   ]
+      // [ a2 ] = [  00.0  00.0  00.5  00.0  00.0  00.0 ][ P(i+1)   ]
+      // [ a1 ] = [  00.0  01.0  00.0  00.0  00.0  00.0 ][ P'(i+1)  ]
+      // [ a0 ] = [  01.0  00.0  00.0  00.0  00.0  00.0 ][ P''(i+1) ]
```

Comment:
The leading zeroes make these hard to read. Can we right-align but left-pad with spaces?